sdk/trace/provider.go

```diff
@@ -350,6 +354,25 @@ func WithSpanLimits(sl SpanLimits) TracerProviderOption {
 	})
 }
 
+func applyTracerProviderEnvConfigs(cfg *tracerProviderConfig) {
+	for _, opt := range tracerProviderOptionsFromEnv() {
+		opt.apply(cfg)
+	}
+}
+
+func tracerProviderOptionsFromEnv() []TracerProviderOption {
+	var opts []TracerProviderOption
+
+	sampler, err := samplerFromEnv()
+	if err != nil {
+		otel.Handle(err)
```

Comment:
If we are going to handle the errors at this level then we should be returning all of our errors back, not handling some of them within the `samplerFromEnv()`.

Reply:
Fixed.